backend/src/controllers/tasks.controller.ts

```diff
@@ -13,9 +21,25 @@ export class TasksController {
 
   @Post()
   insertTask(@Body() task: Task): Promise<void> {
+    if (task.effort < 0) {
+      console.log('Client sent a negative effort value for a new task.');
+      return;
```

Comment:
In order to follow the REST specification and standardize the communication between the frontend and the backend, can we return the HTTP status code `400 Bad Request` if the request body does not pass the validation criteria?